torchci/README.md

```diff
@@ -85,3 +85,13 @@ console.
 2. Save the query, creating a new version.
 3. Download the query with `yarn node scripts/downloadQueryLambda.mjs <workspace> <queryname> <version>`.
 4. Update `rockset/prodVersion.json` with the new version of the lambda.
+
+
+### Modifying Deployment Settings
+
+If you ever need to modify the deployment settings like the oauth callbacks, domain names, there's a few places that you need to change these settings in. Here's a list:
```

Comment:
```suggestion
If you ever need to modify the deployment settings like the oauth callbacks, domain names, there are a few places that you need to change these settings in:
```

nit, but I also have no idea what oauth callbacks and domain names refer to

Reply:
jk i do know what they are, but I think it'd be good to link them to what they're used for, e.g. "domain names (for HUD)" etc.